packages/db/postgres/functions.sql

```diff
@@ -265,16 +271,30 @@ BEGIN
                   c.dag_size
           FROM psa_pin_request psa_pr
           JOIN content c ON c.cid = psa_pr.content_cid
-          JOIN pin p ON p.content_cid = psa_pr.content_cid
           JOIN auth_key a ON a.id = psa_pr.auth_key_id
           WHERE a.user_id = query_user_id::BIGINT
           AND psa_pr.deleted_at is null
-          AND p.status = 'Pinned'
           GROUP BY psa_pr.content_cid,
                   c.dag_size
         ) AS pinned_content), 0)
     );
+  return (total)::TEXT;
+END
+$$;
+
+CREATE TYPE stored_bytes AS (uploaded TEXT, psa_pinned TEXT, total TEXT);
 
+CREATE OR REPLACE FUNCTION user_used_storage(query_user_id BIGINT)
```

Comment:
keeping this function because `users_by_storage_used` uses it. So, just refactored to re-use partial functions created